starter_code/hangman.js

```diff
@@ -20,16 +22,21 @@ class Hangman {
   }
 
   checkClickedLetters(key) {
-      if( !this.letters.includes(key) ){
```

Comment:
Aqui la solucion era un poco mas simple para que pase el test de jasmine, pero como le metiste mas funcionalidad si lo cambias tu juego ya no funciona.

 ` if (this.letters.includes(key)){`
 `  return false`
  `} else {`
  `  return true`
 ` }`